XenModel/Alerts/PerfmonDefinition.cs

```diff
@@ -179,6 +179,7 @@ private PerfmonDefinition(XmlNode parentNode)
                 else if (node.Name.Equals(ALARM_TYPE_ELEMENT_NAME))
                 {
                     name = node.Attributes[ALARM_COMMON_ATTR_NAME].Value;
+                    success = true;

```

Comment:
NB This line is present so that we don't write an erroneous log line at the bottom of this function.